monitoring/dashboards/eigenda-metrics.json

```diff
@@ -572,6 +572,92 @@
       "title": "Batch by status",
       "type": "stat"
     },
+{
+      "datasource": {
+        "type": "prometheus",
+        "uid": "prometheus"
+      },
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "noValue": "Reachability metrics not found",
+          "mappings": [
+            {
+              "options": {
+                "0": {
+                  "color": "red",
+                  "index": 1,
+                  "text": "UNREACHABLE"
```

Comment:
nit: Not a big fan of all caps text. Just `Unreachable` and `Online` should do?

Reply:
I'll stop yelling